packages/codec/lib/interface/decoders/contract.ts

```diff
@@ -405,7 +511,14 @@ export class ContractInstanceDecoder {
     }
   }
 
-  //input is similar to watchMappingKey; will unwatch all descendants too
+  /**
+   * Opposite of watchMappingKey; unwatches the specified mapping key.  See watchMappingKey
+   * for more on how watching mapping keys works, and on how the parameters work.
+   * Note that unwatching a mapping key will also unwatch all its descendants.
+   * E.g., if m is of type mapping(uint => mapping(uint => uint)), then unwatching
+   * m[0] will also unwatch m[0][0], m[0][1], etc, if these are currently watched.
+   * This function has the same caveats as watchMappingKey.
```

Comment:
Probably worth using a codeblock and inline code spans? I dunno, your call.

Reply:
Actually doing this, I didn't find a code block useful.